src/tools/core.ts

```diff
@@ -24,12 +24,12 @@ function filterProjectsByName(projects: ProjectInfo[], projectNameFilter: string
 function configureCoreTools(server: McpServer, tokenProvider: () => Promise<AccessToken>, connectionProvider: () => Promise<WebApi>) {
   server.tool(
     CORE_TOOLS.list_project_teams,
-    "Retrieve a list of teams for the specified Azure DevOps project.",
+    "Retrieve a list of teams for the specified Azure DevOps project. Use pagination (top/skip) for large projects. The Azure DevOps API may limit results to ~720 teams even with higher 'top' values - use 'skip' to get additional teams.",
```

Comment:
is it a good idea to explicitly mention pagination and how top/skip interact in the tool description?